beef

```diff
@@ -31,6 +31,13 @@ if RUBY_PLATFORM.downcase.include?('mswin') || RUBY_PLATFORM.downcase.include?('
   exit 1
 end
 
+#
+# @note Ask user if they would like to update beef
+#
+print "Would you like to check and download the latest BeEF update? y/n: " 
+response = gets
+system("git pull && bundle") if response.strip == 'y' 
```

Comment:
We might not want to use system.
There are two options use the ruby-git gem or use [open3](https://docs.ruby-lang.org/en/2.0.0/Open3.html)

Reply:
done, removed 'system' good call